test/end-to-end/account_statement/account_statement.spec.js

```diff
@@ -35,14 +35,15 @@ describe('Account Statement Core', () => {
     AccountStatement.expectRowCount(0);
   });
 
-  it.skip(`comment the rows for account ${sample.account} with ${sample.comment}`, () => {
-    // select the first row
-    AccountStatement.selectRow(0);
+  // skip throw error : Test `title` should be a "string" but "function" was given instead.
```

Comment:
:dancer: Any particular reason why this had to be commented?

Reply:
On Windows, the `skip` function in `it.skip(` break the e2e test down by throwing this error : **Test `title` should be a "string" but "function" was given instead**

Reply:
Is your mocha version up to date?